submissions/nataliaArtsiukh/OOP Exercise/index.js

```diff
@@ -0,0 +1,100 @@
+class Habitant {
+  constructor(name, gender, saying, friends, legs) {
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+    this.legs = legs;
+  }
+  toString() {
+    let {name, gender, saying, friends, legs} = this;
```

Comment:
`const` is more approriate here. `let` implies (and creates expectation) we are going to change the value.